scripts/install.tpl.sh

```diff
@@ -38,5 +39,10 @@ kubectl_wait "cert-manager" "deployment/cert-manager-webhook" "60s"
 kubectl apply -f "https://github.com/operator-framework/catalogd/releases/download/${catalogd_version}/catalogd.yaml"
 kubectl_wait "olmv1-system" "deployment/catalogd-controller-manager" "60s"
 
+if [[ "${install_default_catalogs,,}" != "false" ]]; then
```

Comment:
I thought this made more sense than checking for true since we want the default to be installing the catalogs.